ApiDoctor.Validation/OData/Transformation/SchemaConfigFile.cs

```diff
@@ -63,6 +68,12 @@ public class SchemaConfig
         /// </summary>
         [JsonProperty("supportedTags")]
         public string[] SupportedTags { get; set; }
+
+        /// <summary>
+        /// optionally specify workloads whose errors should be treated as warnings
+        /// </summary>
+        [JsonProperty("treatErrorsAsWarningsWorkloads")]
```

Comment:
`treatErrorsAsWarningsForFilesContaining`

Reply:
I can rename the property, but rename the jsonproperty specification later because its named like that in the docs already and would need a new Pr to change.